build.gradle

```diff
@@ -175,7 +175,12 @@ ext {
         'platformJvm', 'languageJvm', 'languageJava', 'languageGroovy', 'languageScala',
         'platformNative', 'platformPlay', 'idePlay', 'languageNative', 'ideNative', 'testingBase',
         'testingNative', 'testingJvm', 'pluginDevelopment', 'pluginUse', 'resourcesHttp',
-        'resourcesSftp', 'resourcesS3', 'toolingApiBuilders', 'compositeBuilds', 'buildCacheHttp'
+        'resourcesSftp', 'resourcesS3', 'compositeBuilds', 'buildCacheHttp'
+    ].collect {
+        project(it)
+    }
+    extensionPluginProjects = [
+            'toolingApiBuilders'
```

Comment:
I think that a lot more projects fall into this "internal extensions" category, e.g. `pluginUse`. It might be an interesting follow-up to remove some of these purely internal things from `gradleApi()` as well.